container-search/src/main/java/com/yahoo/search/dispatch/SearchCluster.java

```diff
@@ -67,15 +69,16 @@
     // Only needed until query requests are moved to rpc
     private final FS4ResourcePool fs4ResourcePool;
 
-    public SearchCluster(DispatchConfig dispatchConfig, FS4ResourcePool fs4ResourcePool,
```

Comment:
Refactoring comment: I think SearchCluster is becoming large enough that it makes sense to create a dispatch.searchcluster sub-package and move the nested classes inside SearchCluster to it.